tools/ja-rule/USBDeviceManager.cpp

```diff
@@ -180,7 +183,7 @@ void USBDeviceManager::SignalEvent(EventType event, JaRuleWidget* widget) {
     return;
   }
 
-  if (pthread_equal(m_start_thread_id, Thread::Self())) {
+  if (m_in_start && pthread_equal(m_start_thread_id, Thread::Self())) {
```

Comment:
Because on Windows pthread_t is a struct ( https://github.com/OpenLightingProject/ola/blob/master/include/ola/thread/Thread.h#L39 ), we can't assign the int zero to it. If you're happy that just initialising it again with pthread_t() will be safe enough, we can drop it, the Internet seemed to suggest a separate bool would be the safest option.